client/js/main.js

```diff
@@ -24,10 +24,21 @@ function getValidDropTargets(widget) {
 
     let isValid = true;
     for(const key in t.get('dropTarget')) {
-      if(widget.get(key) != t.get('dropTarget')[key] && (key != 'type' || widget.get(key) != 'deck' || t.get('dropTarget')[key] != 'card')) {
-        isValid = false;
-        break;
+      if(Array.isArray(t.get('dropTarget')[key])) {
+        let count = 0;
```

Comment:
Perhaps this whole branch could be more clearly rewritten as
```
        let thisKeyValid = false;
        for (let i = 0; i < (t.get('dropTarget')[key]).length; i++) {
          if((t.get('dropTarget')[key]).includes(widget.get(key))) {
            thisKeyValid = true;
            break;
          }
        }
        if (!(isValid = thisKeyValid))
          break;
      }
```

Reply:
Yeah, I think that is better.  It brings it back to true/false, and breaks as soon one value matches.